corehq/apps/cloudcare/views.py

```diff
@@ -471,19 +471,13 @@ def get_fixtures(request, domain, user_id, fixture_id=None):
 
     assert user.is_member_of(domain)
     restore_user = user.to_ota_restore_user()
-    if not fixture_id:
-        ret = ElementTree.Element("fixtures")
-        for fixture in generator.get_fixtures(restore_user):
-            ret.append(fixture)
```

Comment:
This would not worked with cached fixtures, which are returned as a UTF-8 encoded string.